src/Features/CSharp/Portable/UsePatternMatching/CSharpAsAndNullCheckCodeFixProvider.cs

```diff
@@ -102,5 +117,64 @@ public MyCodeAction(Func<CancellationToken, Task<Document>> createChangedDocumen
             {
             }
         }
+
+        private class FirstStatementTracker
+        {
+            private readonly Collection<StatementSyntax> _removed;
+            public ICollection<StatementSyntax> FirstStatements { get; }
+
+            public FirstStatementTracker()
+            {
+                _removed = new Collection<StatementSyntax>();
+                FirstStatements = new Collection<StatementSyntax>();
+            }
```

Comment:
constructor can be completely eliminated, and these fields can be initialized insitu.